cpg_workflows/jobs/picard.py

```diff
@@ -132,7 +132,16 @@ def markdup(
     resource = HIGHMEM.request_resources(ncpu=4)
     # enough for input BAM and output CRAM
     resource.attach_disk_storage_gb = 250
-    resource.set_to_job(j)
+
+    # check for a memory override for impossible samples
+    if (memory_override := get_config()['workflow'].get('exceptional_picard')) is not None:
```

Comment:
Should this config name be a bit more explicit?
```suggestion
    if (memory_override := get_config()['workflow'].get('exceptional_markdup_mem_gb')) is not None:
```